app/views/admin/users/index.html.slim

```diff
@@ -0,0 +1,23 @@
+section.section
+  .columns
+    .column
+      h1.title Slack Users
+
+      table.table.is-hoverable
+        thead
+          tr
+            th Admin
+            th First Name
+            th Last Name
+            th Created At
+            th Email
+            th Timezone
+        tbody
+          - @users.each do |user|
+            tr[class = (user.is_deleted? ? "has-text-grey-light" : "")]
+              td= (user.is_admin ? "✓" : "")
```

Comment:
Might be nice to add a helper method for the admin section something like `check(user.is_admin)` which would do the above. Imagine we'll do this a lot for things like this.